plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/StandardProjectWizard.java

```diff
@@ -20,12 +39,18 @@
 
 public class StandardProjectWizard extends Wizard implements INewWizard {
 
-  private AppEngineStandardWizardPage page;
+  private WizardPage page;
   private AppEngineStandardProjectConfig config = new AppEngineStandardProjectConfig();
+  private boolean hasAppEngineComponent;
 
   public StandardProjectWizard() {
     this.setWindowTitle("New App Engine Standard Project");
-    page = new AppEngineStandardWizardPage();
+    hasAppEngineComponent = doesAppEngineJavaComponentExist();
+    if (hasAppEngineComponent) {
```

Comment:
Those are very good ideas. We definitely want to make this as easy on and obvious to the user as possible. That's going to take quite a bit more work, and probably several PRs so I'd like to move ahead with this much and iterate.